libminifi/src/utils/net/AsioSocketUtils.cpp

```diff
@@ -72,18 +72,18 @@ bool AsioSocketConnection::connectTcpSocketOverSsl() {
   asio::error_code err;
   asio::ip::tcp::resolver::results_type endpoints = resolver.resolve(socket_data_.host, std::to_string(socket_data_.port), err);
   if (err) {
-    logger_->log_error("Resolving host '%s' on port '%s' failed with the following message: '%s'", socket_data_.host, std::to_string(socket_data_.port), err.message());
+    logger_->log_error("Resolving host '{}' on port '{}' failed with the following message: '{}'", socket_data_.host, std::to_string(socket_data_.port), err.message());
```

Comment:
all these `std::to_string()`s in this file could be removed

Reply:
:+1: https://github.com/apache/nifi-minifi-cpp/pull/1670/commits/6a5aebd7790dbf83d5418c03b6168ae3a0b6bf38#diff-1c49ef68c8086eafeeba05f851885876f763cbb327cc8103c6a3a5d8460246a8L75